x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/advanced_editor.scss

```
@@ -3,4 +3,8 @@
   @include euiFontSizeS;
   min-height: $euiSizeXL;
   width: 100%;
+}
+
+.lnsRangesOperation__popoverNumberField {
+  width: 100px;
```

This is a good opportunity to use the `ch` unit of measurement. 

> Represents the width, or more precisely the advance measure, of the glyph "0" (zero, the Unicode character U+0030) in the element's font.

```suggestion
  width: 14ch; // Roughly 10 characters plus extra for the padding
```